javatests/arcs/core/entity/DummyVariableEntity.kt

```diff
@@ -0,0 +1,52 @@
+package arcs.core.entity
+
+import arcs.core.data.*
+
+class DummyVariableEntity : VariableEntityBase(ENTITY_CLASS_NAME, SCHEMA), Storable {
+    var text: String? by SingletonProperty()
+    var ref: Reference<DummyEntity>? by SingletonProperty()
+    var bools: Set<Boolean> by CollectionProperty()
+    var nums: Set<Double> by CollectionProperty()
+
+    private val nestedEntitySpecs = mapOf(
+        DummyEntity.SCHEMA_HASH to DummyEntity
+    )
+
+    fun getSingletonValueForTest(field: String) = super.getSingletonValue(field)
```

Comment:
why not use `getSingletonValue(field)` directly?

Reply:
I'm not sure; I'm just following the conventions from the other class. 

I can revert it here -- but then I'd like to change the other class for consistency.

Reply:
... given the other comment below: I'm just going to remove these superfluous methods.